src/commandHandlers/bio.ts

```diff
@@ -13,7 +13,7 @@ import { log } from '../logger';
 export const command = async (arg: string, embed: MessageEmbed, message: Message) => {
     const args = arg.split('||');
     const mention = message.mentions.users.first();
-
+    args[0] = args[0].toLowerCase();
```

Comment:
Looks good, but I think creating a new `const` here would be better

Reply:
okay, let me do that!